apps/frontend/src/app/5_pages/AavePage/components/BorrowPositionsList/components/EfficiencyModeCard/components/DisableEModeForm/DisableEModeForm.tsx

```diff
@@ -0,0 +1,169 @@
+import { formatReserves, formatUserSummary } from '@aave/math-utils';
+
+import React, { FC, useCallback, useMemo } from 'react';
+
+import { t } from 'i18next';
+
+import {
+  Button,
+  ErrorBadge,
+  ErrorLevel,
+  Icon,
+  IconNames,
+  SimpleTable,
+  SimpleTableRow,
+} from '@sovryn/ui';
+import { Decimal } from '@sovryn/utils';
+
+import { AmountRenderer } from '../../../../../../../../2_molecules/AmountRenderer/AmountRenderer';
+import { config } from '../../../../../../../../../constants/aave';
+import { useAaveSetUserEMode } from '../../../../../../../../../hooks/aave/useAaveSetUserEMode';
+import { useAaveUserReservesData } from '../../../../../../../../../hooks/aave/useAaveUserReservesData';
+import { translations } from '../../../../../../../../../locales/i18n';
+import { EModeCategory } from '../../../../../../../../../types/aave';
+import { AaveCalculations } from '../../../../../../../../../utils/aave/AaveCalculations';
+import { CollateralRatioHealthBar } from '../../../../../CollateralRatioHealthBar/CollateralRatioHealthBar';
+
+type DisableEModeFormProps = {
+  current: EModeCategory;
+  onComplete: () => void;
+};
+
+export const DisableEModeForm: FC<DisableEModeFormProps> = ({
+  current,
+  onComplete,
+}) => {
+  const { handleDisableUserEMode } = useAaveSetUserEMode();
+  const { userReservesData, reservesData, timestamp } =
+    useAaveUserReservesData();
+
+  const onConfirm = useCallback(() => {
+    handleDisableUserEMode({ onComplete });
+  }, [handleDisableUserEMode, onComplete]);
+
+  const newSummary = useMemo(() => {
+    if (!userReservesData || !reservesData) {
+      return;
+    }
+
+    const {
+      marketReferenceCurrencyDecimals,
+      marketReferenceCurrencyPriceInUsd: marketReferencePriceInUsd,
+    } = reservesData.baseCurrencyData;
+    return formatUserSummary({
+      userEmodeCategoryId: 0, // disable mode
+      currentTimestamp: timestamp,
+      marketReferencePriceInUsd,
+      marketReferenceCurrencyDecimals,
+      userReserves: userReservesData.userReserves,
+      formattedReserves: formatReserves({
+        currentTimestamp: timestamp,
+        marketReferencePriceInUsd,
+        marketReferenceCurrencyDecimals,
+        reserves: reservesData.reservesData,
+      }),
+    });
```

Comment:
move to `normalizeEmodeSummary` in .utils.tsx and return a already formatted version instead of doing inline multiplications down below